aws-api/src/main/java/com/amplifyframework/api/aws/AppSyncGraphQLRequestFactory.java

```diff
@@ -102,6 +102,15 @@ private AppSyncGraphQLRequestFactory() { }
     static <T extends Model> GraphQLRequest<T> buildQuery(
             Class<T> modelClass,
             QueryPredicate predicate
+    ) throws ApiException {
+        return buildQuery(modelClass, predicate, DEFAULT_QUERY_LIMIT, null);
+    }
+
+    static <T extends Model> GraphQLRequest<T> buildQuery(
```

Comment:
IMO everything going on in this `AppSyncGraphQLRequestFactory` is fair game, in this plugin.